.github/workflows/doc.yml

```diff
@@ -39,5 +39,3 @@ jobs:
         with:
           name: docs
           path: out/doc
-      - name: Test
-        run: NODE=$(command -v node) make test-doc-ci TEST_CI_ARGS="-p actions --measure-flakiness 9"
```

Comment:
The title of this workflow starts with "Tests and ..." we need to remove the tests part